docs/02-install.md

```diff
@@ -144,6 +151,92 @@ file in your installation artifacts directory and simply re-export it:
 
     export KUBECONFIG=/path/to/something/auth/kubeconfig
 
+# Advanced Installations
+While the OpenShift 4 installer's purpose in life is to streamline operations
+in order to guarantee success, there are a few options that you can adjust by
+using a configuration file. Namely, you can change the default number of
+instances of workers and masters, and you can change both the master and
+initial worker EC2 instance types.
+
+## Generate Installer Configuration File
+This example will use the `--dir` option. The first step is to ask the
+installer to pre-generate an installer configuration file,
+`install-config.yaml`:
+
+```sh
+openshift-install --dir /path/to/something create install-config
+```
+
+After following the same interactive prompts you saw earlier when performing
+a basic/default installation, this will generate a file called
+`install-config.yaml` in the folder `/path/to/something`. The important
+stanzas in the file to examine are the two `machines` stanzas:
+
+```yaml
+machines:
+- name: master
+  platform: {}
+  replicas: 3
+- name: worker
+  platform: {}
+  replicas: 3
+```
+
+The platform sections are empty, deferring per-platform implementation
+decisions for these machines to the installer. There is also a platform
+section at the bottom of the file. That section is for per-platform cluster
+config for non-machine properties.
+
+Let's modify our config file to specify that we want 6 initial workers, all
+of size `c5.xlarge`. Again, the only section to modify is the `machines`
+section:
+
+```yaml
+machines:
+- name: master
+  platform: {}
+  replicas: 3
+- name: worker
+  platform: 
+    aws:
+      type: c5.xlarge
+  replicas: 6
+```
+
+Then, as before, run the installer with the `--dir` option:
+
+    openshift-install --dir /path/to/something create cluster
+
+The installer will notice the `install-config.yaml` and not prompt you for
+any input. It will simply begin to perform the installation. When you get to
+the exercises for scaling/exploring your cluster, note the starting machine
+types and quantities.
+
+### NOTE
+It's also possible to change the volume configuration for the EC2 instance.
+In the case of large or busy clusters, tuning the volume parameters to get
+more IOPS for etcd may improve performance and stability. The example below
+shows changing both the instance type and the root volume for the masters:
+
+```YAML
+machines:
+- name: master
+  platform:
+    aws:
+      type: m5.large
+      rootVolume:
+        iops: 700
```

Comment:
ques: how did we choose these values? users might just reuse these values and it would be nice if these are based on some internal recommendations?

Reply:
These values were provided by someone else as examples. They are not for any particular/specific use case, but I don't know.